pyweek/challenge/templates/challenge/index.html

```diff
@@ -161,11 +161,27 @@
 <p>
 Entries must be developed in Python during the challenge, and must
 incorporate some theme decided at the start of the challenge. See the <a
-href="{{ STATIC_URL }}rules.html">challenge rules</a> for more information.
+href="https://pyweek.readthedocs.io/en/latest/rules.html">challenge rules</a> for more information.
 </p>
     </div>
     <div class="col-md-6">
         {% include "motd.html" %}
+
+        <div style="background-color: #7289DA; margin: 1em 0; padding: 1em 1em 0.8em; border-radius: 6px; color: white">
+
+            <a href="https://discord.gg/python">
+            <img src="{{STATIC_URL}}/python-discord.png" alt="Python Discord"
+                style="float: left; margin: -5px 0 0 -8px"
+            >
+            </a>
+            <p><a
+                href="https://discord.gg/python"
+                style="color: white; text-decoration: underline"
+                >
+                Join us in Discord</a>
```

Comment:
```suggestion
                Join us on Discord</a>
```

I think usually the preposition here would be `on`.